python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -320,6 +332,28 @@ def __statisticsSummary( plug ) :
 		"layout:section", "Camera",
 	],
 
+	# Purpose
+
+	"options.includedPurposes" : [
+
+		"description",
+		"""
+		Limits the objects included in the render according to the values of their `usd:purpose`
+		attribute.
```

Comment:
Might be nice to include a sentence here describing the influence "Default" has on locations without a `usd:purpose` attribute?

Reply:
Added in https://github.com/GafferHQ/gaffer/pull/5470/commits/bf593aa9de398f06c19887edac0d66887bf4bc96.